example/Components/SimpleForm.js

```diff
@@ -28,27 +28,47 @@ const Form = ({
 (
   <div className='form'>
     <div className='formGroup'>
-      <label>Name</label>
+      <label htmlFor='name'>Name</label>
       <input
         type='text'
+        id='name'
+        aria-errormessage='name-error'
+        aria-invalid={isValid(errors.name) ? null : 'true'}
         className={isValid(errors.name) ? '' : 'error'}
         name='name'
         value={form.name}
         onChange={updateValueAndValidate}
       />
-      <div className='errorPlaceholder'>{ createErrorMessage(errors.name) }</div>
+      <div 
+        id='name-error' 
+        className='errorPlaceholder'
+        aria-live='assertive' 
+        aria-atomic='true'
+        aria-relevant='additions text'>
+          <a href='#name'>{ createErrorMessage(errors.name) }</a>
```

Comment:
having the error message appear in a link with a bookmark `href` to the input's id creates an extra affordance for keyboard users to help focus the invalid field.